source/includes/properties/_01-create.md

```diff
@@ -0,0 +1,11 @@
+## Create
+
+```shell
+curl -d "value=PropertyValue" http://goserver.com:8153/go/properties/PipelineName/1243/StageName/1/JobName/MyPropertyName
```

Comment:
See comment about quoting above, also the `-d` argument goes at the end, like this - http://localhost:4567/#update-an-agent